report/main.qmd

```diff
@@ -0,0 +1,139 @@
+      \renewcommand{\arraystretch}{1} % Increase row spacing
+      \renewcommand\thefigure{\arabic{figure}}
+    fontsize: 12pt
+    colorlinks: true
+bibliography: ../reference/ref.bib
+---
+
+```{=tex}
+\thispagestyle{empty}
+\clearpage
+\pagenumbering{arabic}
+```
+# Introduction
+
+Zero-calorie beverages offer an alternative to sugary drinks that can help to avoid the harmful effects of artificial sweeteners. Therefore, it is important to understand what actions can motivate people to switch to these products. This study aims to assess the impact of messaging and discounts on the buying behaviour of zero-calorie and sugary drinks. The study’s statistical advice outlines how to quantitatively evaluate four research questions on this topic. The first question to be addressed is whether these interventions lead to increased consumption of zero-calorie beverages and decreased consumption of sugary beverages. Subsequently, the study will also investigate whether the effects vary across different sites (second question) and whether combining interventions results in larger effects (third question). Lastly, the study will compare the effectiveness of calorie-equivalent messaging with simple calorie admonishment (fourth question).
+
+# Data Description and Summaries
+
+To evaluate the effects of interventions, the study gathered data on beverages sold at four cafeterias and three convenience stores across three sites. The dataset recorded daily sales of these beverages for a period of 221 days and summarizes the data by site. Nevertheless, the observations for each site start on different dates: site A starts on day 1, site B on day 14, and site C on day 20.[^1] In total, there are 631 observations in the dataset.
+
+[^1]: We are using the client's submission names for sites, but they may not correspond to the same sites in the submission. Here, A corresponds to chop, B to NF and C to NS.
+
+The dataset includes variables related to time, sales, site and intervention. The time variables are the count of days since the start of the study and the day of the week. The sales variables include zero-calorie, sugary, 100% juice, orange juice, sports and total beverages sold, but only zero-calorie and sugary beverages are considered for this analysis. As for site and intervention, there is a variable for each one. @tbl-vars summarizes the variables available in the dataset, their classification, and how they are measured.
+
+|          Variable           |    Type     |
+|:---------------------------:|:-----------:|
+| Day of the quasi-experiment | Continuous  |
+|       Day of the week       | Continuous  |
+|            Site             | Categorical |
+|        Intervention         | Categorical |
+|    Sugary beverages sold    | Continuous  |
+| Zero-calorie beverages sold | Continuous  |
+|    Other beverages sold     | Continuous  |
+
+: **Description of variables** {#tbl-vars tbl-colwidths="\[35,25\]"}
+
```

Comment:
Day of the qusi-experiment seems to be ordinal, and day of the week seems to be either ordinal or categorical. Also, probably elaborate on why you say that the missing values "do not affect this analysis". Is it because the missing values are on variables that we are not interested in, or the data are missing at random so the result won't be biased if we simply delete them, or the missing values are during follow-up period so our analysis on interventions won't be affected?